packages/language-server/src/completion/completions.json

```diff
@@ -94,7 +105,7 @@
     {
       "label": "@@id([])",
       "documentation": "Defines a multi-field ID on the model.",
-      "fullSignature": "@id(_ fields: FieldReference[])",
+      "fullSignature": "@@id(_ fields: FieldReference[])",
```

Comment:
This change seems unrelated?

Reply:
I thought this was an unrelated bug fix - the `fullSignature` was just buggy before.

Reply:
+1 to @Jolg42 's comment